src/index.ts

```diff
@@ -42,6 +44,17 @@ export {
   flush,
 }
 
+export namespace CliUx {
```

Comment:
In testing, this caused some compilation issues when accessing `CliUx.Table` and `CliUx.Config`

I was able to fix it by removing the namespace and adding this to the exports on line 21, `cliUx as CliUx`